context/AuthContext.tsx

```
@@ -0,0 +1,282 @@
+      const newChild = {
+        id: Date.now().toString(),
+        name,
+        age,
+        avatar,
+        parentId: user.id,
+      };
+
+      await AsyncStorage.setItem("child", JSON.stringify(newChild));
+      setChild(newChild);
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)